### PR TITLE
Combined test coverage

### PR DESCRIPTION
**Pull request by prateekdefi**

### CODE REVIEW

**Review by arjun-io — DISMISSED** [review]
Awesome

**Comment by prateekdefi** [comment]
I am looking in why CI is not triggered for this PR.

**Comment by kbrizzle** [comment]
I don't see the `perennial-verifier` package under any of these, am I missing it somewhere or is that one missing?

**Comment by prateekdefi** [comment]
> I don't see the `perennial-verifier` package under any of these, am I missing it somewhere or is that one missing?

It is missing. `perennial-deploy` is also missing. These workspaces were not a part of the CI earlier as well so I ignored them. Would you like me to add these? @kbrizzle

**Comment by arjun-io** [comment]
> > I don't see the `perennial-verifier` package under any of these, am I missing it somewhere or is that one missing?
> 
> It is missing. `perennial-deploy` is also missing. These workspaces were not a part of the CI earlier as well so I ignored them. Would you like me to add these? @kbrizzle 

All packages except for perennial-deploy should be added 👍

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -465,11 +477,208 @@ jobs:
...
+      - name: Install lcov
+        run: sudo apt-get update && sudo apt-get install lcov
+      - name: Combine test coverage
+        run: lcov -a periphery_unit_coverage/periphery_unit_coverage.info -a periphery_integration_coverage/periphery_integration_coverage.info --output-file combined_coverage.info
+      - name: 'Periphery Combined Test Code Coverage Report'
         uses: romeovs/lcov-reporter-action@v0.3.1
         with:
-          title: '[Extensions] Integration Test Coverage Report'
-          lcov-file: ./packages/perennial-extensions/coverage/lcov.info
+          title: '[Periphery] Test Coverage Report'
```
Comment (arjun-io) [inline .github/workflows/CI.yml:680]: very tiny nit - can we add "Combined" before "Test" here for uniformity 😅

**Review by arjun-io — APPROVED** [review]

**Review by EdNoepel — APPROVED** [review]
Appreciate the whitespace between sections for readability, and that Verifier tests are now run by CI.

**Pull request merged**